content/kb/troubleshooting-failure-to-delete-orphaned-pod-volume-directory.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Troubleshooting: Failure to delete orphaned Pod volume directory"
+author: Ray Chang
+draft: false
+date: 2023-03-23
+categories:
+  - "csi"
+---
+
+## Applicable versions
+
+All Longhorn versions.
```

Comment:
NIT: This heading is probably most often used to talk about Longhorn version (as you have done), but it might also be a good location to highlight Kubernetes version.
```suggestion
All Longhorn versions.
Kubernetes versions before v1.27.
```

Reply:
Thanks. I've updated to include the K8S version information in that block.